src/components/forms/Input/__tests__/Input-test.js

```diff
@@ -129,4 +130,16 @@ describe('Input', function() {
     setValidCallback(false);
     expect(element.state.valid).toBe(false);
   });
+
+  it("animates label to large hint when 'animateLabel' is true", function() {
+    var element = TestUtils.renderIntoDocument(<Input animateLabel={ true } />);
+    findByClass(element, 'Input__label--empty-blur');
+  });
+
+  it("label is small if has value", function() {
+    var element = TestUtils.renderIntoDocument(<Input animateLabel={ true } value="foo" />);
+    var input = scryByClass(element, 'Input__label--empty-blur');
+
+    expect(input.length).toBeLessThan(1);
```

Comment:
Nope. For some strange reason. If it does not find anything it returns an object instead of an array. So length is undefined.